### PR TITLE
Supplementaries 翻译更新

### PR DESCRIPTION
**Pull request by ChuijkYahus**

### CODE REVIEW

File: projects/1.19/assets/supplementaries/supplementaries/lang/zh_cn.json [hunk 1]
```diff
@@ -274,18 +276,18 @@
   "block.supplementaries.soap_block": "肥皂块",
   "item.supplementaries.soap": "肥皂",
   "item.supplementaries.soap_water": "肥皂水",
-  "advancements.story.wrench.title": "工程师游戏中",
-  "advancements.story.wrench.description": "使用扳手旋转一个方块",
+  "advancements.husbandry.wrench.title": "工程师游戏中",
+  "advancements.husbandry.wrench.description": "使用扳手旋转一个方块",
```
Comment (RCqaq) [inline projects/1.19/assets/supplementaries/supplementaries/lang/zh_cn.json:280]: `Turn Table`应该是`转台`。
`How the turn tables` 
Michael Scott's way of saying how the tables have turned. It is used when you are trying to reference The Office, or if you are simply being stupid. -摘自UrbanDictionary
出处可以在油管搜到，意思大概是“说都不会话了”。

File: projects/1.19/assets/supplementaries/supplementaries/lang/zh_cn.json [hunk 2]
```diff
@@ -274,18 +276,18 @@
...
-  "filled_map.antique": "古式地图",
+  "filled_map.antique.tooltip": "褪色",
+  "filled_map.antique": "褪色地图",
   "block.supplementaries.globe_sepia": "墨绘地球仪",
   "block.supplementaries.bubble_block": "肥皂泡",
   "advancements.nether.goblet.title": "来杯鸡尾酒",
   "advancements.nether.goblet.description": "从酒杯中喝东西",
-  "advancements.story.turn_table.title": "转台好玩吗",
-  "advancements.story.turn_table.description": "用转台旋转你自己",
+  "advancements.husbandry.turn_table.title": "转台好玩吗",
```
Comment (RCqaq) [inline projects/1.19/assets/supplementaries/supplementaries/lang/zh_cn.json:289]: `How the turn tables`
Michael Scott's way of saying how the tables have turned. It is used when you are trying to reference The Office, or if you are simply being stupid. -摘自UrbanDictionary
出处可以在油管搜到，意思大概是“说都不会话了”。

Comment (ChuijkYahus) [reply projects/1.19/assets/supplementaries/supplementaries/lang/zh_cn.json:289]: 改成`转台转到晕`怎么样

Comment (ChuijkYahus) [reply projects/1.19/assets/supplementaries/supplementaries/lang/zh_cn.json]: 感谢@HaloSense 在群内的建议（）
采用`风水流流转`

**Comment by ChuijkYahus** [comment]
顺便更新了一遍120的汉化

File: projects/1.20/assets/supplementaries/supplementaries/lang/zh_cn.json [hunk 3]
```diff
@@ -245,7 +246,7 @@
   "block.supplementaries.present_yellow": "黄色礼物盒",
   "block.supplementaries.present_orange": "橙色礼物盒",
   "block.supplementaries.present_red": "红色礼物盒",
-  "block.supplementaries.present_pink": "粉红色礼物盒",
+  "block.supplementaries.present_pink": "粉色礼物盒",
```
Comment (RCqaq) [inline projects/1.20/assets/supplementaries/supplementaries/lang/zh_cn.json:249]: 为什么变成粉色了？

File: projects/1.20/assets/supplementaries/supplementaries/lang/zh_cn.json [hunk 4]
```diff
@@ -325,7 +326,7 @@
   "block.supplementaries.trapped_present_yellow": "黄色陷阱礼物盒",
   "block.supplementaries.trapped_present_orange": "橙色陷阱礼物盒",
   "block.supplementaries.trapped_present_red": "红色陷阱礼物盒",
-  "block.supplementaries.trapped_present_pink": "粉红色陷阱礼物盒",
+  "block.supplementaries.trapped_present_pink": "粉色陷阱礼物盒",
```
Comment (RCqaq) [inline projects/1.20/assets/supplementaries/supplementaries/lang/zh_cn.json:329]: 同上

File: projects/1.20/assets/supplementaries/supplementaries/lang/zh_cn.json (hunk 3 above)
Comment (ChuijkYahus) [reply projects/1.20/assets/supplementaries/supplementaries/lang/zh_cn.json:249]: 怪，已改

**Review by dovisutu — APPROVED** [review]

**Review by luoxueyasha — DISMISSED** [review]

File: projects/1.20/assets/supplementaries/supplementaries/lang/zh_cn.json [hunk 5]
```diff
@@ -425,13 +426,24 @@
...
-  "message.supplementaries.tattered": "残破",
+  "message.supplementaries.tattered": "破烂不堪",
   "painting.supplementaries.bombs.title": "Bombs",
   "painting.supplementaries.bombs.author": "Plantkillable",
   "item.supplementaries.cartographers_quill": "制图羽毛笔",
   "message.supplementaries.cartographers_quill": "未知",
-  "message.supplementaries.quill_failed": "制图羽毛笔搜索失败",
-  "message.supplementaries.quill_finished": "制图师地图已绘制完毕",
+  "message.supplementaries.quill_failed": "制图羽毛笔无法在附近找到目标。",
+  "message.supplementaries.quill_retry": "制图羽毛笔无法在附近找到目标。试试看在其他地方使用。",
```
Comment (luoxueyasha) [inline projects/1.20/assets/supplementaries/supplementaries/lang/zh_cn.json:435]:
```suggestion
  "message.supplementaries.quill_retry": "制图羽毛笔无法在附近找到目标。在其他地方使用试试。",
```
感觉“看”有点多余

**Review by luoxueyasha — APPROVED** [review]

**Comment by luoxueyasha** [comment]
@ChuijkYahus 这个pr还更新不，我怕被继续背刺（

**Comment by ChuijkYahus** [comment]
不更了

**Pull request merged**